x/sequencer/types/messages.go

```diff
@@ -81,6 +84,22 @@ func (msg *MsgCreateSequencer) ValidateBasic() error {
 		if !ok {
 			return sdkerrors.Wrapf(sdkerrors.ErrInvalidType, "Expecting cryptotypes.PubKey, got %T", pk)
 		}
+
+		var err error
+
+		// parse to secp256k1 or edc25519 pubkey and checks validity (the func includes cryptographic validation)
+		switch pk.Type() {
+		case "secp256k1":
+			_, err = secp256k1.ParsePubKey(pk.Bytes())
+		case "edc25519":
```

Comment:
only ed25519 expected for dymint key

Reply:
i added both in case this changes in the future, but can be removed

Reply:
I think it should be removed

Reply:
i removed it, but then i think we should prevent creating sequencers with non ed25519 keys, which i think now is possible (but we should open a new issue for that)